src/TemplateWrapper.php

```diff
@@ -42,6 +42,11 @@ public function display(array $context = [])
         $this->template->display($context, \func_get_args()[1] ?? []);
     }
 
+    public function stream(array $context = []): iterable
```

Comment:
To declare a return type compatible with the type expected by `StreamedResponse`.
```suggestion
    /**
     * @return iterable<string>
     */
    public function stream(array $context = []): iterable
```